code/src/components/GamePage.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { fetchNext } from 'reducers/game';
+import { StartPage } from 'components/StartPage';
+
+export const GamePage = () => {
+  const dispatch = useDispatch();
+
+  const descriptionText = useSelector((store) => store.game.description);
+  const coordinates = useSelector((store) => store.game.coordinates);
+  const actionsObject = useSelector((store) => store.game.actions);
+  const user = useSelector((store) => store.game.user);
+
+  let content;
+
+  if (user === null) {
+    content = <StartPage />;
+  } else {
+    content = (
+      <div>
+        <h2 className='description-container nes-container'>
+          {descriptionText}
+        </h2>
+        <p className='coordinates'>Your coordinates are: {coordinates}</p>
+        <h3 classname='play-text'>Choose your direction wisely...</h3>
```

Comment:
Hide this on the last page of the game maybe?

Reply:
Yes 🤓